### ISSUE TITLE
v1.12.5 : Do you have any plans for the next release?

### ISSUE THREAD

**Issue by kakakakakku** [deepmap/oapi-codegen#1019]
Hi there!

Do you have any plans for the next release?

 I noticed that there is an error in the `README.md` file regarding the behavior of `x-enum-varnames`. I know that although the master branch has the newest code generator with https://github.com/deepmap/oapi-codegen/pull/880, it has not yet been released.

Therefore, the behavior of the latest version `v1.12.5` is different from what is described in the `README.md` file. Of course I know notice below.

>⚠️ This README may be for the latest development version, which may contain unreleased changes. Please ensure you're looking at the README for the latest release version.

```yaml
components:
  schemas:
    Object:
      properties:
        category:
          type: integer
          enum: [0, 1, 2]
          x-enum-varnames:
            - notice
            - warning
            - urgent
```

To summarize the differences, I have listed them below:

## If I use `v1.12.4` version

```sh
$ oapi-codegen -version
github.com/deepmap/oapi-codegen/cmd/oapi-codegen
v1.12.4
```

```go
// Defines values for ObjectCategory.
const (
	N0 ObjectCategory = 0
	N1 ObjectCategory = 1
	N2 ObjectCategory = 2
)
```

## If I use `master` branch version

```sh
$ oapi-codegen -version
github.com/deepmap/oapi-codegen/cmd/oapi-codegen
v1.12.5-0.20230403173426-fd06f5aed350
```

```go
// Defines values for ObjectCategory.
const (
	Notice  ObjectCategory = 0
	Urgent  ObjectCategory = 2
	Warning ObjectCategory = 1
)
```

Thanks!

**Comment by brayansdt** [comment]
UP

**Comment by cynsupercat** [comment]
Hello, same issue here. External $refs do not work well with 12.1.4.

**Comment by rha1-godaddy** [comment]
+1 for me as well

**Comment by shineli1984** [comment]
Having the same issue.

**Comment by jamietanna** [comment]
Sorry folks, should be released in the next few weeks https://github.com/deepmap/oapi-codegen/discussions/1072

In the period between releases you're welcome to pin to a commit SHA off our default branch - one of the handy things I love about Go 😁
